core/src/main/java/com/linecorp/armeria/common/auth/OAuth1aTokenBuilder.java

```diff
@@ -125,13 +137,60 @@ public OAuth1aTokenBuilder version(String version) {
     }
 
     /**
-     * Sets additional (or user-defined) parameters.
+     * Sets additional parameters. If the specified {@code additionals} has pre-defined parameters,
+     * the parameters are set automatically to this builder and removed from the {@code additionals}.
      */
     public OAuth1aTokenBuilder additionals(Map<String, String> additionals) {
```

Comment:
This method is not only for additional parameters but any parameters including pre-defined ones. Perhaps we should rename it to `putAll()`? We could also add methods like `put(CharSequence, CharSequence)`.

Reply:
Thanks, added `put()` and `putAll()`. I didn't use `CharSequence` though because I feel it's too much.
If it's necessary, jut let me know. 😉

Reply:
SGTM. Thanks, @minwoox